### PR TITLE
feat: dashboard breakpoints 

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Added responsiveness to the dashboard page, now the page adjusts correctly on different screen sizes. 
Jira Ticket: https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-337

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Visual tests only.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/components/BCHeader/index.tsx [hunk 1]
```diff
@@ -2,52 +2,121 @@ import React from 'react';
...
 } from '@carbon/react';
+import { Link, useNavigate } from 'react-router-dom';
 import {
   Search,
   Notification,
   UserAvatar
 } from '@carbon/icons-react';
-import { Link, useNavigate } from 'react-router-dom';
+
+import * as Icons from '@carbon/icons-react';
```
Comment (RMCampos) [inline src/components/BCHeader/index.tsx:22]: Is that a good idea? Just wondering.

File: src/components/BCHeader/styles.scss [hunk 2]
```diff
@@ -15,6 +19,35 @@ a.header-link:link {
...
+button.#{vars.$bcgov-prefix}--header__action.#{vars.$bcgov-prefix}--header__menu-trigger:focus {
+  border-color: colors.$white;
+}
+
+button.#{vars.$bcgov-prefix}--header__action.#{vars.$bcgov-prefix}--header__action--active svg {
+  fill: colors.$gray-100;
+}
+
+/* Small - Up to 320px */
+@media only screen and (max-width: 672px) {
```
Comment (RMCampos) [inline src/components/BCHeader/styles.scss:49]: Up to 320px? Or 672px?

**Review by RMCampos — APPROVED** [review]
Great job!!

File: src/components/BCHeader/index.tsx (hunk 1 above)
Comment (mgaseta) [reply src/components/BCHeader/index.tsx:16]: It is not hahaha, already fixed it.

File: src/components/BCHeader/styles.scss (hunk 2 above)
Comment (mgaseta) [reply src/components/BCHeader/styles.scss:49]: 672, the commentary is wrong, already fixed it.

**Pull request merged**